spec/fixtures/blueprints.lua

```diff
@@ -116,7 +143,6 @@ function _M.new(db)
     }
   end)
 
-  local upstream_name_seq = new_sequence("upstream-%d")
```

Comment:
I think those lines are for a better reading experience so perhaps we should't modify that
Any particular reason that we moved these to the top of the function?

Reply:
In my feel,  it is helpful to reuse and to avoid declaring some of them repeatedly in this way.

Reply:
Discussed offline, this is to also align with the code that we have in `blueprints.lua` in enterprise version, because in enterprise version we have multiple reuse of these sequence, but not in community version.